x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -141,7 +141,23 @@ export class MBMapContainer extends React.Component {
 
   _getFeaturesUnderPointer(mbLngLatPoint) {
     const mbLayerIds = this._getMbLayerIdsForTooltips();
-    return this._mbMap.queryRenderedFeatures(mbLngLatPoint, { layers: mbLayerIds });
+    const PADDING = 2;//in pixels
+    const mbBbox = [
+      {
+        x: mbLngLatPoint.x - PADDING,
+        y: mbLngLatPoint.y - PADDING
+      },
+      {
+        x: mbLngLatPoint.x + PADDING,
+        y: mbLngLatPoint.y + PADDING
+      }
+
+    ];
+
+    if (!this._mbMap) {
+      return [];
+    }
```

Comment:
Small optimization, you could move this conditional return to the top of the method so we can skip the tooltip method call and the padding calcs.